### DISCUSSION TITLE
Adding Kourtney and Shannon to the access

### DISCUSSION THREAD

**Discussion by cberchok** [nmfs-ost/PAM-OpenScience#11] [Q&A]
@Kourtney-Burger - I know you said you added in instrx on how to do this, but I can't find where those are.  

I've figured out how to get to the 'add people' button, but what role am I assigning you and Shannon?

**Comment by Kourtney-Burger** [comment]
Is this for adding people to your lab manual? 

You should add at least one more person at your lab as an admin (they have full access and permissions). I think Shannon asked to be added to yours so we could figure out the bug of you not seeing changes on the website but it sounds like that has been resolved?
 
![image](https://github.com/user-attachments/assets/1eb846b1-f94e-4ec3-b468-51d7ed4731d8)

**Comment by shannonrankin** [comment]
Yes, that is correct--
Add either/both of us if you need us to fix problems (but this one fixed
itself?),
and add at least one other AFSC person as an admin (the 'hit by a bus'
backup)

:)
Shannon

**Comment by cberchok** [comment]
add you guys as admin then; I think with my track record I will do that!

I have added the hit-by-bus backups already!  

Thanks guys!

**Reply by Kourtney-Burger** [reply]
Sounds good!
